src/App.css

```diff
@@ -10,10 +10,49 @@ li {
 
 header,
 form,
-nav > ul,
-.bookStyle {
+nav > ul {
   display: flex;
   justify-content: space-around;
   align-items: center;
   background-color: #f0f0f0;
 }
+
+.book {
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.sectionOne,
+.bookDetails {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.buttons {
+  width: 250px;
+  display: flex;
+  justify-content: space-evenly;
+  flex-direction: row;
+  margin-bottom: 20px;
+}
+
+.button {
+  border: 1px solid #66f;
+  border-radius: 8px;
+  padding: 10px;
+}
+
+.book button {
+  padding: 7px;
+  color: #999;
+  background-color: #5f5;
+  border-radius: 10px;
+}
+
+.sectionTwo {
+  padding-top: 50px;
+}
```

Comment:
- [ ] As was suggested by the previous reviewer please follow the given design on [zeplin]() which means colors, layouts, etc
<img width="1422" alt="Screen Shot 2022-11-12 at 18 28 16" src="https://user-images.githubusercontent.com/44745855/201484239-e7e857a6-e729-42c8-8067-861b79014ab4.png">